deploy/examples/multus-validation.yaml

```diff
@@ -0,0 +1,99 @@
+#################################################################################################################
+# We highly recommend validating your Multus configuration before you install Rook.
+# This job aims to automate that operation by using the validation tool. Run this job after
+# installing the rook operator and before installing any Custom Resources.
+#################################################################################################################
+---
+# Aspects of multus validation job that require access to the operator/cluster namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps", "configmaps/finalizers", "pods"]
+    verbs: ["get", "create", "update", "delete"]
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources: ["network-attachment-definitions"]
+    verbs: ["get"]
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["get", "list", "delete"]
```

Comment:
These definitely aren't all the privileges the serviceaccount will need. It'll also need the ability to get,create,delete daemonsets and get,list,create,delete,deletecollection deployments. Probably list pods also.

@Nikhil-Ladha am I wrong, or have you not been running the job in a development environment to see if it works?

Reply:
I have been running the job in a dev env to confirm it's working, and it is then when I am facing the above mentioned SCC issue.
Regarding the above privileges, they _maybe_ required, but the thing is that the job is stuck at the SCC issue, so I never actually found out if these were also needed.